mlir/lib/Conversion/MemRefToEmitC/MemRefToEmitC.cpp

```diff
@@ -132,12 +132,15 @@ struct ConvertGlobal final : public OpConversionPattern<memref::GlobalOp> {
                                          "cannot convert result type");
     }
 
-    bool staticLinkage =
+    // We are explicit in specifier the linkage because the default linkage
+    // for constants is different in C and C++.
+    bool staticSpecifier =
```

Comment:
Are you sure we don't just want to match `Private/Public` for now and leave `Nested` as unsupported?

Reply:
ok, will do